frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -763,17 +756,16 @@ function NoNodeContextMenuOptions(props: NoNodeContextMenuProps): ItemType[] {
     infoRows,
     allowUpdate,
   } = props;
-  const dispatch = useDispatch();
-  const isAgglomerateMappingEnabled = useSelector(hasAgglomerateMapping);
-  const isConnectomeMappingEnabled = useSelector(hasConnectomeFile);
 
-  const isProofreadingActive = useSelector(
-    (state: OxalisState) => state.uiInformation.activeTool === AnnotationToolEnum.PROOFREAD,
-  );
+  const state = Store.getState();
+  const isAgglomerateMappingEnabled = hasAgglomerateMapping(state);
+  const isConnectomeMappingEnabled = hasConnectomeFile(state);
+
+  const isProofreadingActive = state.uiInformation.activeTool === AnnotationToolEnum.PROOFREAD;
 
-  useEffect(() => {
-    dispatch(maybeFetchMeshFilesAction(visibleSegmentationLayer, dataset, false));
-  }, [visibleSegmentationLayer, dataset]);
+  // useEffect(() => {
```

Comment:
comment can be removed now.

Reply:
Good catch. I mean to ask, the `useEffect(...)` call has two dependencies and will be executed when these change. I was not completely clear if the next code covers this.

Reply:
Done